.gitattributes

```diff
@@ -1,2 +1,3 @@
 typings/**/*.pyi linguist-generated=true
+tests/unit/_internal/bento/testdata/**/*.yaml linguist-generated=true
```

Comment:
these YAML files should probably not be linguist-generated=true

Reply:
fixed